x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -295,15 +295,6 @@ export const ActionForm = ({
     </SectionLoading>
   ) : (
     <>
-      <EuiTitle size="s">
```

Comment:
Removing this affects the actions form for creating rules within the stack rule create/edit flyout. If you don't want this to show up in the detections rule creation form, maybe we can make it configurable whether it shows up?

Reply:
Hey, https://elastic.slack.com/archives/C03259SFNP8/p1663871840518189?thread_ts=1663774333.737969&cid=C03259SFNP8 Maybe this would give you more context. The general idea was to put actions and response actions on the ''equal'' level, because otherwise we just saw Throttling Dropdown, and Response Actions. Of course if we set a throttle, then Actions pops out, but that wasn't really logical. 

The result:
![Zrzut ekranu 2022-09-22 o 20 34 37](https://user-images.githubusercontent.com/16632552/192006791-50030b1a-c5fa-46f6-a201-2d9c0396d56f.png)

However, I didn't know that this form is being used somewhere else, my IDE doesn't show that really. Do you mind showing me where exactly that TItle was being used? 
I'll be happy to adjust, sorry if I broke any other view by the way.

Reply:
https://github.com/elastic/kibana/pull/141041/commits/0900c7cb020425f54c58847aa063d2db1e8b76a6 Does this make sense?